CMakeLists.txt

```diff
@@ -294,39 +291,12 @@ if(WIN32)
     set(CMAKE_SHARED_LIBRARY_PREFIX "")
     set(CMAKE_SHARED_MODULE_PREFIX "")
 
-    set(CSOUND_WINDOWS_LIBRARIES
-        advapi32
-        comctl32
-        comdlg32
-        glu32
-        kernel32
-        odbc32
-        odbccp32
-        ole32
-        oleaut32
-        shell32
-        user32
-        uuid
-        winmm
-        winspool
-        ws2_32
-        wsock32)
+    set(CSOUND_WINDOWS_LIBRARIES kernel32 ws2_32)
 
     if(MSVC)
         # Experimented with flags but did not make change performance results
         #set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Oi /fp:fast /arch:AVX2")
         #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Oi /fp:fast /arch:AVX2")
-
-        # Replace the default CRT linkage from dynamic (MD) to static (MT)
-        if(STATIC_CRT STREQUAL "ON")
```

Comment:
This one was a bit of a concern. I remember there being a critical problem when not static linking CRT. I checked my email history and in 2017 there was a thread on csound developer mailing list (not sure if that's archived somewhere online; the title of the thread was "Windows: Problem again with libsndfile (critical)") and it pointed to https://github.com/libsndfile/libsndfile note about CRT linking. The context was that using the Windows build at the time with Blue on Windows had it where no sound files were loading. 

I'm a bit concerned about the changes in this PR and dynamic/static linking. Have the builds from this branch been tested with Host apps? (i.e., with Blue, or Cabbage used as a VST within a DAW?)

Reply:
@kunstmusik This code was actually not used as it was added by me and controlled by a CMake flag `STATIC_CRT`. I was experimenting with it before but like you said, I remember that issue so never really used it. Anyway, this can be controlled by VCPKG anyways. In this branch I'm using the `x64-windows-static-md` target triplet which contains this:
```
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE static)
```

So that means x64 arch, dynamic CRT and the libraries are then linked statically. The only change from before is that the Csound dependencies will be linked statically but they're all using a dynamic CRT.